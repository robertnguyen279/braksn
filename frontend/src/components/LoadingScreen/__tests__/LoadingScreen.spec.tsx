import React from 'react';
import { shallow } from 'enzyme';

import LoadingScrene from '../LoadingScreen';

describe('LoadingScreen', () => {
  it('should render correctly', () => {
    const component = shallow(<LoadingScrene />);
    expect(component).toMatchSnapshot();
  });
});
