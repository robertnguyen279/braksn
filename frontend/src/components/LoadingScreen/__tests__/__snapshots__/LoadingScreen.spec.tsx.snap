// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen should render correctly 1`] = `
<div
  className="absolute h-screen inset-0 w-screen flex items-center justify-center"
>
  <Loading />
</div>
`;
